src/Products/Viewer/Controllers/ViewerApiController.cs

```diff
@@ -674,6 +674,7 @@ private static LoadDocumentEntity GetDocumentPages(PostedDataEntity postedData,
             // get/set parameters
             string documentGuid = postedData.guid;
             string password = string.IsNullOrEmpty(postedData.password) ? null : postedData.password;
+            if (!File.Exists(documentGuid)) { throw new Exception("File not found."); }
```

Comment:
Codacy found an issue: ['System.Exception' should not be thrown by user code.](https://app.codacy.com/gh/groupdocs-viewer/GroupDocs.Viewer-for-.NET-MVC/pullRequest?prid=5958476)